### ISSUE TITLE
Add unit tests

### ISSUE THREAD

**Issue by ankitkumarr** [Azure/azure-functions-tooling-feed#266]
This repository holds feed for several packages and tools, which some of our other tools (VS, VS Code Extension) rely on.

Even though the feed is updated via automation (not yet for v4 feed), we don't have tests to validate the feed. We need to add some unit tests that validate basic scenarios for our feed such as -- making sure the AppSettings values are correct, capabilities are accurate and never remove unintentionally, and several other scenarios that typically shouldn't change.

cc: @phil-allen-msft (thanks for suggesting this)

**Comment by fabiocav** [comment]
This needs requirements defined. @ankitkumarr can we discuss with @soninaren

**Comment by ankitkumarr** [comment]
From our discussion, we concluded that we need a small E2E test project, that looks at the `cli-feed-v4.json` (and maybe others) and perform validations against the release entries pointed to by the `tags` in the JSON.

This project would run in the CI triggered by any changes made to the feed.

Moving this to the next Sprint for potentially picking up and implementation.

**Comment by fabiocav** [comment]
Area SME: @soninaren

**Comment by tommck** [comment]
It would be REALLY great if you at LEAST had a test to see if the feed was serving up valid JSON (which it is not doing right now) so that the tooling would at least function in vscode.

You can add a million other requirements later, but you should at least add the bare minimum right away

**Comment by tommck** [comment]
@alexweininger - maybe you can get them to do a basic JSON parse test in CI so #315 doesn't happen again?

**Comment by alexweininger** [comment]
> @alexweininger - maybe you can get them to do a basic JSON parse test in CI so #315 doesn't happen again?

Yeah I agree, a basic test that at least ensures the feed file is valid should be implemented.

**Comment by tommck** [comment]
Looks like there's an attempt in the build.ps1 file here: https://github.com/Azure/azure-functions-tooling-feed/blob/cad8f58d536f4949465506d4ecc94fcf6f8cb822/build.ps1#L19

But I assume powershell's JSON conversion is more forgiving
